charts/opensearch/values.yaml

```diff
@@ -548,3 +548,13 @@ serviceMonitor:
   # labels:
   #  k8s.example.com/prometheus: kube-prometheus
   labels: {}
+
+  # Basic Auth configuration for the service monitor
+  # basicAuth:
+  #   username:
+  #     secretName: credentials
+  #     secretKey: username
+  #   password:
+  #     secretName: credentials
+  #     secretKey: password
+  basicAuth: {} 
```

Comment:
Thanks @eyenx can you please some details in the README ? also can `basicAuth` have one secret with username and password ?

Reply:
I will take a look at it tomorrow

Reply:
I now added a `existingSecret` functionality and a way to deliver the `username` & `pw` over the helm charts values